src/compile/data/sample.ts

```diff
@@ -1,4 +1,4 @@
-import {SampleTransform as VgSampleTransform} from 'vega-typings';
+import {SampleTransform as VgSampleTransform} from 'vega';
```

Comment:
It's probably worth centralizing aliasing for transform. 

Basically do 

`export {BinTransform as VgBinTransform, ...} from 'vega';`

so we don't have to do alias in other files.


This can be done later though (e.g., when @invokesus refactor to use vega typings for all transforms).